EnRonSwanson/BangazonCLI
# Mb mgr methods

## Fixes - Method ticket

## Changes proposed in this pull request: 
- Creating New Product method
- Program/CLI/Table Statement edits to support Creating New Product
- Active Customer class/manager to get and set who is "logged in"
- minor ERD edit added from our discussion last Friday

## How to test: 
- pull branch into test one
- delete db file if exists
- dotnet run
- ensure fields are written under correct column headings in DB browser.

@EnRonSwanson/
